packages/renderer/src/lib/preferences/docker-compat/PreferencesDockerCompatibilityRendering.svelte

```diff
@@ -1,10 +1,17 @@
 <script lang="ts">
 import SettingsPage from '../SettingsPage.svelte';
+import PreferencesDockerCompatibilitySocketMappingStatus from './PreferencesDockerCompatibilitySocketMappingStatus.svelte';
 </script>
 
 <SettingsPage title="Docker compatibility">
   <div slot="subtitle" class="text-sm font-thin mt-2">
```

Comment:
Styling should be removed here, doesn't match the Resources page. (or if design wants it changed, we should do it in SettingsPage.

Reply:
pushed a change